### PR TITLE
Upgrade to 4.9.0

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "4.8.6" %}
+{% set version = "4.10dev" %}
```
Comment (isuruf) [inline recipe/meta.yaml]: How about 4.9.0? We don't like packaging dev versions unless there's a convincing argument to do so.

**Comment by saraedum** [comment]
Ok. I did not know that.

**Comment by saraedum** [comment]
@isuruf: I'm a bit confused about the `build.sh`. Was all this necessary to build gap-4.8.6? I have to admit I did not understand all of this but at least I could not find the files referenced there in my local clone/build of gap.

**Comment by saraedum** [comment]
Nevermind. I think I started with the wrong sources. The ones at github are quite different from the ones at their homepage.

**Comment by saraedum** [comment]
Let me close this mess. I just realized that 4.9.0 has not been officially released on their homepage https://www.gap-system.org/Releases/index.html.

**Comment by alex-konovalov** [comment]
@saraedum yes, that was a beta release for package authors! glad that this was figured out.

**Pull request closed without merging**